### PR TITLE
Cloud native corfu: adding missing tools/scripts

### PR DESCRIPTION
**Pull request by xnull**
## Overview

Description:

The changes in this patch help us to deploy a Cloud-native Corfu on Kubernetes: 
1. adding missing tools/scripts into Corfu's docker image
2. infra maven target now depends on cmdlets - we need it to bootstrap cloud corfu
3. some fixes made to cmdlets - retries for pings, tls enabled

Why should this be merged: 

Related issue(s) (if applicable): #<number>

## Checklist (Definition of Done):

- [x] There are no TODOs left in the code
- [x] [Coding conventions](https://github.com/CorfuDB/CorfuDB/wiki/Corfu-Style-Guidelines) (e.g. for logging, unit tests) have been followed
- [x] Change is covered by automated tests
- [x] Public API has Javadoc

### CODE REVIEW

File: infrastructure/pom.xml [hunk 1]
```diff
@@ -29,6 +29,85 @@
...
+                                <phase>package</phase>
+                                <goals>
+                                    <goal>exec</goal>
+                                </goals>
+                                <configuration>
+                                    <executable>cp</executable>
+                                    <arguments>
+                                        <argument>../cmdlets/target/cmdlets-${project.version}-shaded.jar</argument>
+                                        <argument>target</argument>
+                                    </arguments>
```
Comment (xcchang) [inline infrastructure/pom.xml:92]: looks like the path needs to be corrected

Comment (xnull) [reply infrastructure/pom.xml]: fixed

Comment (xnull) [reply infrastructure/pom.xml]: fixed 2-nd time

**Review by PavelZaytsev — APPROVED** [review]

File: infrastructure/Dockerfile [hunk 2]
```diff
@@ -1,10 +1,20 @@
...
+ARG CORFU_TOOLS_JAR
+
 WORKDIR /app
 
 RUN apk add --update iptables sudo
 
+COPY target/${CORFU_JAR} /usr/share/corfu/lib/${CORFU_JAR}
+COPY target/${CORFU_TOOLS_JAR} /usr/share/corfu/lib/${CORFU_TOOLS_JAR}
+
+COPY target/bin /usr/local
```
Comment (hisundar) [inline infrastructure/Dockerfile:13]: shouldn't this be `/user/share/corfu/bin` ?

Comment (xnull) [reply infrastructure/Dockerfile]: fixed

**Review by hisundar — APPROVED** [review]

**Review by hisundar — APPROVED** [review]
we'll keep adding more scripts as we bring more functionality into corfudb instead of in the clients

**Pull request merged**